### PR TITLE
[Mailer] Add documentation for Mailjet mailer

### PR DESCRIPTION
**Pull request by tcheymol**
Documenting the introduction of the Mailjet mailer component ref: [symfony/symfony#36541 (comment)](https://github.com/symfony/symfony/pull/36541)

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -47,6 +47,7 @@ MailChimp           ``composer require symfony/mailchimp-mailer``
 Mailgun             ``composer require symfony/mailgun-mailer``
 Postmark            ``composer require symfony/postmark-mailer``
 SendGrid            ``composer require symfony/sendgrid-mailer``
+Mailjet             ``composer require symfony/mailjet-mailer``
```
Comment (noniagriconomie) [inline mailer.rst:50]: i think they are listed alphabeticaly by purpose

File: mailer.rst [hunk 2]
```diff
@@ -86,16 +87,17 @@ transport, but you can force to use one:
...
+==================== ============================================= =========================================== ========================================
+ Amazon SES           ses+smtp://ACCESS_KEY:SECRET_KEY@default     ses+https://ACCESS_KEY:SECRET_KEY@default   ses+api://ACCESS_KEY:SECRET_KEY@default
+ Google Gmail         gmail+smtp://USERNAME:PASSWORD@default       n/a                                         n/a
+ Mailchimp Mandrill   mandrill+smtp://USERNAME:PASSWORD@default    mandrill+https://KEY@default                mandrill+api://KEY@default
+ Mailgun              mailgun+smtp://USERNAME:PASSWORD@default     mailgun+https://KEY:DOMAIN@default          mailgun+api://KEY:DOMAIN@default
+ Postmark             postmark+smtp://ID:ID@default                n/a                                         postmark+api://KEY@default
+ Sendgrid             sendgrid+smtp://apikey:KEY@default           n/a                                         sendgrid+api://KEY@default
+ Mailjet              mailjet+smtp://ACCESS_KEY:SECRET_KEY@default n/a                                         mailjet+api://ACCESS_KEY:SECRET_KEY@default
```
Comment (noniagriconomie) [inline mailer.rst:99]: same as above

File: mailer.rst (hunk 1 above)
Comment (tcheymol) [reply mailer.rst]: @noniagriconomie Thx

File: mailer.rst (hunk 2 above)
Comment (tcheymol) [reply mailer.rst]: @noniagriconomie Thx

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Thibaut.

**Pull request merged**